bt_automata/base/neuron.py

```diff
@@ -121,7 +121,7 @@ def sync(self):
             self.set_weights()
 
         # Always save state.
-        self.save_state()
+        #self.save_state()
```

Comment:
What happens to neurons that want to save their state?

Reply:
I dont think they would be missing much:

    def save_state(self):
        bt.logging.warning(
            "save_state() not implemented for this neuron. You can implement this function to save model checkpoints or other useful data."
        )